core/src/components/cat-scrollable/cat-scrollable.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'cat-scrollable',
+  styleUrl: 'cat-scrollable.css',
+  shadow: true,
+})
+export class CatScrollable {
+
+  /** Flags to enable/disable scroll shadows. */
+  @Prop()
+  shadow: { x?: boolean; y?: boolean; } = { x: true, y: true };
+
+  /** Flags to enable/disable overflow. */
+  @Prop()
+  overflow: { x?: boolean; y?: boolean; } = { y: true };
```

Comment:
Please move this  `{ x?: boolean; y?: boolean; }`  in another file.

Reply:
same here